_sass/base.scss

```diff
@@ -4864,6 +4864,15 @@ sup {
   }
 }
 
+.volunteers{
```

Comment:
@Muskan284 thanks for looking into this quickly! What I meant was adding a separate section like what I have below in the css file and then having the anchor contents there:

```css
////////////////////////////////////////////
////////////// Page — Volunteer/////////////
////////////////////////////////////////////
```
The code you have works great 💯 . Can you just add a section immediately after the [Page - Get Involved](https://github.com/hotosm/hotosm-website/blob/gh-pages/_sass/base.scss#L4794-L4796) (and right above the [Page-Donate](https://github.com/hotosm/hotosm-website/blob/gh-pages/_sass/base.scss#L4907-L4909)) and have your `anchor` code moved to the new section?